config.wyam

```diff
@@ -12,7 +12,11 @@ Settings[DocsKeys.BlogRssPath] = "blog/feed/rss/index.xml";
 
 Pipelines.InsertBefore(Docs.Code, "Addins",
     ReadFiles("../addins/*.yml"),
-    Yaml()
+    Yaml(),
+    Meta("Version", string.IsNullOrEmpty(@doc.String("NuGet")) 
```

Comment:
Yes, that was my initial strategy but I struggled to find a "File.Exists" like API in Wyam. Do you know how it can be done or if it is ok to use File.Exists, which would mean we're "breaking out" of the Wyam file system abstractions?

Reply:
@devlead helped me along. I'll push a fix for the problem later.